app.js

```diff
@@ -11,7 +11,7 @@ const bodyParser = require('body-parser')
 
 const app = express();
 
-app.set('view engine','pug');
+app.set('view engine','ejs');
```

Comment:
why are we changing view engine, and what's the difference between the two: `pug` and `ejs'?

Reply:
We use ejs because ejs support vanila javascript code or script in html page itself . in pug we need to minimize the whole html script into pug format .. So we can built application faster on ejs than pug .

You can refer this link for difference : https://www.slant.co/versus/184/185/~ejs_vs_pug-jade